src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -128,6 +129,8 @@ describe('<CourseRunEnrollment />', () => {
 
     const button = screen.getByRole('button', { name: 'Enroll now' });
 
+    const spyConsole = jest.spyOn(ErrorHandler, 'handle').mockImplementation(() => {});
```

Comment:
IMO spyOn was relevant to spy the `error` method of `console`. Since `handle` is the only method exposed by the `handle` module, we could fully mock the module with `jest.mock('utils/errors/handle')`. Then we could simply import `handle` method and test it has been called.

May be you will have to reset mock before each tests.